pages/learn/develop/hardware/i2c-and-spi.md

```diff
@@ -49,9 +49,9 @@ To get you started, here is an [example][i2c-example] that uses i2c to communica
 
 ### SPI
 
-SPI is enabled by default and should work out of the box with the [spi node module][spi-npm].
+SPI is enabled by default on {{ $names.os.lower }}.
```

Comment:
Maybe we can provide a bit context what it means enabled by default? some users might want to disable also? `dtparam=spi=on` in `config.txt`. Perhaps link to https://www.balena.io/docs/reference/OS/advanced/#setting-device-tree-overlays-dtoverlay-and-parameters-dtparam?